vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java

```diff
@@ -2293,6 +2293,43 @@ public int getPvScsiDeviceControllerKey() throws Exception {
         throw new Exception("VMware Paravirtual SCSI Controller Not Found");
     }
 
+    protected VirtualSCSIController getScsiController(DiskControllerType type) {
+        switch (type) {
+            case pvscsi:
+                return new ParaVirtualSCSIController();
+            case lsilogic:
+                return new VirtualLsiLogicController();
+            case lsisas1068:
+                return new VirtualLsiLogicSASController();
+            case buslogic:
+                return new VirtualBusLogicController();
+            default:
+                return new VirtualLsiLogicController();
+        }
```

Comment:
```
switch (type) {
             case pvscsi:
                 return new ParaVirtualSCSIController();
             case lsisas1068:
                 return new VirtualLsiLogicSASController();
             case buslogic:
                 return new VirtualBusLogicController();
             case lsilogic:
             default:
                 return new VirtualLsiLogicController();
}
```

Reply:
@nvazquez check and address if this is ok.

Reply:
Done, thanks